views/js/controller/Delivery/monitoring.js

```diff
@@ -516,6 +518,96 @@ define([
                 }
 
                 /**
+                 * Formats a timer
+                 * @param {Number} totalSeconds
+                 * @returns {String}
+                 */
+                function formatTime(totalSeconds) {
```

Comment:
You have access to the `moment` library in this module : http://momentjs.com/ please use it's formatting features